src/realm/object-store/sync/sync_session.cpp

```diff
@@ -500,21 +502,20 @@ void SyncSession::handle_error(SyncError error)
     else if (error_code.category() == realm::sync::protocol_error_category()) {
         SimplifiedProtocolError simplified =
```

Comment:
i wish we could get rid of SimplifiedProtocolError.

Reply:
I'll look into it.